pkg/settings/settings.go

```diff
@@ -308,6 +311,37 @@ func (settings *Settings) ShouldInitialism() bool {
 	return !settings.NoInitialism
 }
 
+// HasCustomRename returns if the setting to rename columns with
+// custom rename rules is set.
+func (settings *Settings) HasCustomRename() bool {
+	return settings.CustomColumnRename != ""
+}
+
+const multiSplitter = ","
+const ruleSplitter = ":"
+
+// ParseCustomRenameRules reads a custom rename setting string
+// and returns a map for easier rule access.
+// customRenameRule example: column1:Column2,col10:column10.
```

Comment:
Lets be super explicit about this and mention the format first and then an example?

```
Format: from:to
Example: providerId:providerID
Multiple rules can be specified by comma: column1:Column2,col10:column10.
```